### PR TITLE
Upgrade frontend fixups

### PR DESCRIPTION
**Pull request by jimmcgaw**

### CODE REVIEW

File: src/components/layout/Info.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ const Info = ({ onToggleModalVisibility }) => {
   
   return (
     <div className="info">
-      {/* <OverlayTrigger placement="bottom" overlay={infoTooltip} delayHide={150} delayShow={300}>
-        <FaInfoCircle onClick={onClose} size={25}/>
-      </OverlayTrigger> */}
+      {/* <OverlayTrigger placement="bottom" overlay={infoTooltip} delayHide={150} delayShow={300}> */}
+      {/* </OverlayTrigger> */}
```
Comment (elimisteve) [inline src/components/layout/Info.js:19]: @jimmcgaw Leaving this in as a reminder to replace this?

Comment (jimmcgaw) [reply src/components/layout/Info.js:19]: Aye. Going to see about incrementally upgrading react-bootstrap and see if we can get the same component without compatibility issues with react. If that won't work, then I'll find some other tool in this version to accomplish the same thing.

Comment (elimisteve) [reply src/components/layout/Info.js:19]: Cool, sounds good. I don't know if this is any good, but... it does exist :sweat_smile: https://github.com/ReactTooltip/react-tooltip/

Comment (elimisteve) [reply src/components/layout/Info.js:19]: Additional options: https://reactscript.com/best-tooltip-component/

But using `react-bootstrap` sounds good to me as long as it works well with the latest version of React.

**Pull request merged**